### ISSUE TITLE
Wrong code gen when using $

### ISSUE THREAD

**Issue by davidmartos96** [simolus3/moor#2761]
**Describe the bug**

When a Table schema is defined in Dart and we use $ in the name, the code generation is invalid because Strings that use this name are not escaped or marked as raw strings.

I'm using a tool that generates the drift schema automatically based on a postgres schema and conflicting names include a $ symbol at the end.
 
```dart
class Todo extends Table {
  TextColumn get id => text()();

  TextColumn get listid => text().nullable()();

  TextColumn get text$ => text().named('text').nullable()();

  BoolColumn get completed => boolean()();
}
```

**Comment by simolus3** [comment]
Thanks for the report - I think this is fixed with 5740eb87216703b1995f2779fdbf8b34aab56b5b. Unfortunately it's possible that I've missed some - if it's not too much trouble, could you try to use the [development version](https://drift.simonbinder.eu/docs/internals/) and let me know if that fixes this for you?

**Comment by davidmartos96** [comment]
@simolus3 That did the trick, thank you!
